src/inodes/INodeManager.ts

```diff
@@ -743,14 +743,14 @@ class INodeManager {
     for await (const data of dataDb.createReadStream(options)) {
       // This is to account for the case where a some blocks are missing in a database
       // i.e. blocks 0 -> 3 have data and a write operation was performed on blocks 7 -> 8
-      while (inodesUtils.unbufferId((data as any).key) !== blockCount) {
+      while (inodesUtils.unbufferId((data as any).key) != blockCount) {
         yield Buffer.alloc(blockSize);
       }
-      const plainTextData = await this.db.deserializeDecrypt<string>(
+      const plainTextData = await this.db.deserializeDecrypt(
```

Comment:
Does `raw` being `true` meant that we don't use the generic type parameter? I think it's because we're always expecting a `Buffer` right?

Reply:
Yeah if it is raw then it will always return a buffer